code/__DEFINES/reagents.dm

```diff
@@ -184,12 +184,6 @@
 #define GIN 			"gin"
 #define ABSINTHE 			"absinthe"
 #define RUM 			"rum"
-#define TEQUILA 			"tequila"
-#define VERMOUTH 			"vermouth"
-#define WINE 			"wine"
-#define COGNAC 			"cognac"
-#define HOOCH 			"hooch"
-#define ALE 			"ale"
```

Comment:
It's better not to fix it now because you risk breaking something else and it'll violate atomicity. Nothing wrong with removing the duplicate defines, though.

I have no idea what deadrums are for. Might be poison.